pkg/operation/seed/seed.go

```diff
@@ -1660,3 +1664,43 @@ func cleanupOrphanExposureClassHandlerResources(ctx context.Context, c client.Cl
 
 	return nil
 }
+
+// resizeOrDeleteLokiDataVolumeIfStorageNotTheSame updates the garden Loki PVC if passed storage value is not the same as the current one.
+// Caution: If the passed storage capacity is less than the current one the existing PVC and its PV will be deleted.
+func resizeOrDeleteLokiDataVolumeIfStorageNotTheSame(ctx context.Context, k8sClient client.Client, newStorageQuantity resource.Quantity, log logrus.FieldLogger) error {
```

Comment:
You could add unit tests for this func (I guess you will need to export it first, but to me it seems better to have unit tests).

Reply:
done